recipes-app/node-red-gpio/files/0005-mraa-gpio-dout-Add-D14-D19-for-IOT2050.patch

```diff
@@ -0,0 +1,72 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Baocheng Su <baocheng.su@siemens.com>
+Date: Thu, 28 Dec 2023 10:19:31 +0800
+Subject: [PATCH] mraa-gpio: dout: Add D14-D19 for IOT2050
+
+This was lost during last upstreaming.
+
+Fixes: 25d7d668ef11 ("Add IOT2050 board into mraa Intel nodes (#940)")
+Signed-off-by: Baocheng Su <baocheng.su@siemens.com>
+---
+ hardware/intel/mraa-gpio-dout.html | 10 +++++++++-
+ hardware/intel/mraa-gpio-dout.js   |  5 +++--
+ 2 files changed, 12 insertions(+), 3 deletions(-)
+
+diff --git a/hardware/intel/mraa-gpio-dout.html b/hardware/intel/mraa-gpio-dout.html
+index 8b6e961d5dd2..5ab879887e44 100644
+--- a/hardware/intel/mraa-gpio-dout.html
++++ b/hardware/intel/mraa-gpio-dout.html
+@@ -15,7 +15,7 @@
+         icon: "arrow.png",
+         align: "right",
+         label: function() {
+-            if (this.pin === "14") {
++            if (this.pin === "14" && RED.settings.mraaGpioDoutBoardType !== "IOT2050") {
+                 return "LED";
+             } else {
+                 return this.name||"D"+this.pin;
+@@ -28,6 +28,14 @@
+             if (RED.settings.mraaGpioDoutBoardType === 'Galileo v1') {
+                 $('#node-input-pin').append($("<option></option>").attr("value",14).text("LED - Galileo v1"));
+             }
++
++            if (RED.settings.mraaGpioDoutBoardType === "IOT2050") {
++                for (i of [...Array(20).keys()].slice(14)) {
```

Comment:
Same remark on `for` as above. I think this is very readable.

Reply:
Ack

Reply:
Forgot to update?

Reply:
Apparently changed in a later patch - please do that directly here.

Reply:
I made a mistake when finding the 1st patch, fixed.